front-end/src/screens/ResetPassword/index.tsx

```diff
@@ -21,12 +21,14 @@ interface Props {
 const ResetPassword = ({ className }:Props): JSX.Element => {
 	const router = useRouter();
 	const { token } = router.query;
+	const userId = Number(new URLSearchParams(window.location.search).get('user_id'));
```

Comment:
we should use the useRouter hook for both the token and the user_id.
ps: let's use `userId` rather than `user_id` if possible

Reply:
done